src/main/java/at/hannibal2/skyhanni/utils/LorenzUtils.kt

```diff
@@ -364,18 +369,28 @@ object LorenzUtils {
         return this
     }
 
-    private var lastMessageSent = 0L
+    private var lastMessageSent = SimpleTimeMark.farPast()
+    private val sendQueue: Queue<String> = LinkedList()
+
+    @SubscribeEvent
+    fun sendQueuedChatMessages(ignored: LorenzTickEvent) {
```

Comment:
while the event is ignored all of the existing code still calls the argument event

Reply:
Is that a guideline into which thought was put, or is that simply the convention that grew out of complacency?

Reply:
Convention that doesn't have to be followed. Main review was for the other bit